bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/i18n/mercedesme_de.properties

```diff
@@ -0,0 +1,209 @@
+# binding
```

Comment:
Translations should be provided in Crowdin after the PR has been merged. This will ensure proof-reading by Crowdin translators (PR reviewers are not German proof-readers) and that policies are followed, for example that translations are not merged until 100% completed.

Reply:
I removed translation file from PR